src/components/newsFeed/index.test.js

```diff
@@ -12,7 +12,7 @@ import i18n from '../../i18n';
 import NewsFeed from './index';
 import * as settingsActions from '../../actions/settings';
 
-describe('NewsFeedHOC', () => {
+describe.skip('NewsFeedHOC', () => {
```

Comment:
Why is this skipped? Can we remove the whole file?

Reply:
correct I remove the file